src/main/java/com/revature/pojos/BusinessEmployeeUser.java

```diff
@@ -1,7 +1,12 @@
 package com.revature.pojos;
 
+import javax.persistence.*;
+
+@Entity
+@Table(name="BUSINESS_EMPLOYEE")
 public class BusinessEmployeeUser extends User{
 
+	@Column(name="EMPLOYEE_NAME")
 	private String employerName;
 	
 	private String businessMessage;
```

Comment:
you forgot to add @Column here

Reply:
how would a column go there, is it going to be a join or a relationship column cause the message is stored in another table